### PR TITLE
Charter for review

### LINKED ISSUE #224: Proposed legal & governance improvements for the OpenSSF
There have been some formalizations with governance for projects in the OpenSSF. 
 I *think* these are all for the better, but if you have a concern, *PLEASE* let me know soon! I plan to wait for 1 month (2024-03-12) before starting down this path.

[Here is the full documents](https://docs.google.com/document/d/1mEd3pThttTo6M8n-5mUxIWMe3izQnFRhMVNizhULcrU/edit)

1. *LLC Series*. LF Legal would like this project to become a "series" within "LF Projects, LLC". This creates a legal identity for the project & provides various legal protections. We hope to not *need* legal protections, but it's best to have them. To do that, the TSCs to be listed on the project page on who is in the TSC. 

The TSC then needs to vote on the 
[Technical Charter Draft](https://docs.google.com/open?id=1xi4CN5VpW-SiiMspnKVasmu1o4-L--qIJwmxjL2wHxc)

Make sure the mission is accurate. 

Choose a delegate to sign the 
[Contribution Agreement Draft](https://docs.google.com/open?id=1hKM35BLT2LNBp7XXuLK4NEpilbFZVnb3ldALumtzxkI)

Then the LF will submit the
[Series Agreement Draft](https://docs.google.com/open?id=1GT8mXx0MrR-xqXKEcYZWcwCWkFM90Rg2SMFVAHiHlbc)

Please let me know if you have any questions or concerns.

### PR DESCRIPTION
**Pull request by hythloda**
This PR resolves #224 
The mission specifically needs review.

### CODE REVIEW

**Comment by hythloda** [comment]
@oliverchang has the project had enough time to review the charter? Any thoughts or changes needed? We are hoping to close out the charter.

File: CHARTER.md
Comment (oliverchang) [inline CHARTER.md:10]: To be consistent with https://ossf.github.io/osv-schema/#purpose, could we use "a standard interchange format for describing vulnerabilities in open source packages." verbatim here?

**Comment by hythloda** [comment]
Any other concerns with the charter that stops us from approval?

**Review by oliverchang — COMMENTED** [review]
Sorry for the delay. This generally seems good.

@rsc @chrisbloom7 have been the two other main contributors to this project. Any thoughts?

**Comment by chrisbloom7** [comment]
I have no issue with this 👍🏻

**Comment by hythloda** [comment]
> need to have date charter was adopted entered prior to merging

Working with LF legal, they don't want this charter merged, so there is no date. We are using GitHub in hopes of getting the approval and adopting the date of the last approval, and then we can upload it as a PDF. 
The reason for this is that when it isn't a pdf people have made changes to charters in other groups and then there is the official charter stored with legal and a different one on GitHub.

**Review by oliverchang — APPROVED** [review]

**Comment by oliverchang** [comment]
Apologies for the delay, I thought I already approved it in https://github.com/ossf/osv-schema/pull/228#pullrequestreview-1999964957

**Comment by hythloda** [comment]
> Apologies for the delay, I thought I already approved it in [#228 (review)](https://github.com/ossf/osv-schema/pull/228#pullrequestreview-1999964957)

You indeed have. Do we need approval from @rsc to close it out or can I move this to closed?

**Comment by rsc** [comment]
LGTM thanks.

**Comment by hythloda** [comment]
I am closing this PR as I do not have write access any longer to make the needed changes. The charter that should be added is here: [OSV Schema Technical Charter.pdf](https://github.com/user-attachments/files/16337151/OSV.Schema.Technical.Charter.pdf)

**Pull request merged**
